administrator/language/de-DE/plg_workflow_publishing.ini

```diff
@@ -4,10 +4,10 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_WORKFLOW_PUBLISHING="Workflow - Publishing"
-PLG_WORKFLOW_PUBLISHING_CHANGE_STATE_NOT_ALLOWED="You're not allowed to change the publishing state of this item. Please use a workflow transition."
+PLG_WORKFLOW_PUBLISHING="Workflow – Veröffentlichung"
+PLG_WORKFLOW_PUBLISHING_CHANGE_STATE_NOT_ALLOWED="Es ist nicht erlaubt, den Veröffentlichungsstatus dieses Artikels zu ändern. Bitte verwenden Sie einen Workflow-Übergang."
 PLG_WORKFLOW_PUBLISHING_PUBLISHED="Status: %s"
-PLG_WORKFLOW_PUBLISHING_TRANSITION_ACTIONS_PUBLISHING_DESC="Define the state an item should be set, when executing this transition"
-PLG_WORKFLOW_PUBLISHING_TRANSITION_ACTIONS_PUBLISHING_DO_NO_CHANGE="- Do not change -"
-PLG_WORKFLOW_PUBLISHING_TRANSITION_ACTIONS_PUBLISHING_LABEL="Publishing state"
-PLG_WORKFLOW_PUBLISHING_XML_DESCRIPTION="Add publishing actions to the workflow transitions for your items"
+PLG_WORKFLOW_PUBLISHING_TRANSITION_ACTIONS_PUBLISHING_DESC="Den Status festlegen, in den ein Eintrag gesetzt werden soll, wenn dieser Übergang ausgeführt wird"
```

Comment:
```suggestion
PLG_WORKFLOW_PUBLISHING_TRANSITION_ACTIONS_PUBLISHING_DESC="Die Stufe festlegen, in den ein Eintrag gesetzt werden soll, wenn dieser Übergang ausgeführt wird"
```

Reply:
Vielleicht so (?):
```suggestion
PLG_WORKFLOW_PUBLISHING_TRANSITION_ACTIONS_PUBLISHING_DESC="Den Status festlegen, zu dem ein Eintrag gesetzt werden soll, wenn dieser Übergang ausgeführt wird"
```

Reply:
oder
```suggestion
PLG_WORKFLOW_PUBLISHING_TRANSITION_ACTIONS_PUBLISHING_DESC="Den Status festlegen, auf den ein Eintrag gesetzt werden soll, wenn dieser Übergang ausgeführt wird"
```

Reply:
Statt Status arbeiten wir hier mit Stufe, das war Wunsch gewesen, dass erstmal so zu nennen.

Reply:
Sorry, der "Status" ist ungewollt hereingerutscht.
Was ist mit den Alternativen "in", "zu" oder "auf"?

Reply:
"auf" eine Stufe setzen ist hier wohl das passende.